.github/workflows/ci.yml

```diff
@@ -67,6 +67,28 @@ jobs:
       - name: Running examples
         run: sbt test
         working-directory: examples/event-migration
+
+  publish:
+    needs: [format, test, run-examples]
+    if: github.event_name != 'pull_request'
```

Comment:
Hmmm so `publish` runs on both each `main` build AND git tag build right? does `sbt ci-release` automagically recognize between the two cases (so that it publishes to snapshots or releases)?

Reply:
Yes